README.md

```diff
@@ -4,42 +4,16 @@ As part of the lecture, students have to implement the missing chunks.
 
 ## Setup
 The reference version is Python 3.10. The code is only tested for this version, but should work for any reasonably recent Python version (> 3.5).
-Only one package is required as an external dependency. That is `pytest` which is required to run the (you guessed it) tests.
+Only one package is required as an external dependency. That is `pytest` which is required to run the (you guessed it) tests. To install pytest just run ``pip install pytest`` on your terminal.
 
-Please install pytest only via the Python package manager `pip`. In an own virtual environment this can be achieved with:
+## testing 
```

Comment:
Thanks. But the manual methods should still be described.

Please add your `## testing` section as last section to the README, aka:

## Setup
[...]
## Code and Tasks
[...]

## Test Code
[...]

## Build and Test Scripts
$YOUR_SECTION